test/uport-connect.js

```diff
@@ -1,2 +0,0 @@
-import { Connect, QRUtil, SimpleSigner, Credentials } from '../src/index'
```

Comment:
we need to keep this, this is used for integration testing, all files are copied to the integration-test folder, where there is a different uport-connect.js